Index.html

```diff
@@ -68,6 +68,243 @@ <h2>I’m Hussen Glad to see you!</h2>
+                    </li>
+                     <li class="tag-list">
+                      <h2>css</h2>
+                     </li>
+                     <li class="tag-list">
+                      <h2>javaScript</h2>
+                     </li>
+                 </ul>
+                  <div class="button-acction">
+                      <button class="button-acction-button" type="button"> See Project</button>
+                 </div>
+              </div>
+              </div>
+              </div>
+            </div>
+          </div>
+           <!-- Fourth card  -->
+        <div class="card-work">
+            <div class="card-prtfolio">
+              <img src="img/work-img/Snapshoot Portfolio 4.png" />
+            </div>
+            <div class="work-block">
+              <div class="left-blok">
+                <h2 class="project-title">Tonic</h2>
+                <ul class="frames">
+                  <li class="frames-list">
+                     <h2>CANOPY</h2>
+                    </li>
+                    <li class="frames-list">
+                      <img src="img/work-img/Counter.png" />
+                     </li>
+                     <li class="frames-list">
+                      <h2>Back End Dev</h2>
+                     </li>
+                     <li class="frames-list">
+                      <img src="img/work-img/Counter.png" />
+                     </li>
+                     <li class="frames-list">
+                      <h2>2015</h2>
+                     </li>
+                 </ul>
+                 <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                 
+                 <ul class="tag-block">
+                  <li class="tag-list">
+                     <h2>html</h2>
+                    </li>
+                     <li class="tag-list">
+                      <h2>css</h2>
+                     </li>
+                     <li class="tag-list">
+                      <h2>javaScript</h2>
+                     </li>
+                 </ul>
+                  <div class="button-acction">
+                      <button class="button-acction-button" type="button"> See Project</button>
+                 </div>
+              </div>
+              </div>
+              </div>
+            </div>
+          </div>
+      </section>
+
+      <section class="my-self-container">
+        <div class="top">
+          <div class="text-p">
+            <h2>About
+                Myself</h2>
+          </div>
+          <div class="text-s">
+            <p>
+                Hello I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+          </div>
+
+          <div class="lets-connect-p">
+            <a href="#"><b>LET’S CONNECT</b></a>
```

Comment:
- [ ] I kindly suggest that you ensure that `LET'S CONNECT` title uses an header tag `hx` because it is an header for your social media icons